### PR TITLE
Translate feedback header

### PR DESCRIPTION
**Pull request by abbiefarr**
## Issue

Addresses #415 

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Uses the existing translations for the feedback header. Note that I changed the wording of the Spanish translation because it was too long to fit on the page and was partially hidden behind the Detroit logo.

image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Navigate to the public site and toggle the language setting.

- [x] Desktop View
- [x] Mobile View

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

**Review by willrlin — COMMENTED** [review]
It seems like you're removing the href link from all of the languages except English? Are we not going to have a link at all in there?

**Comment by abbiefarr** [comment]
AFAICT node-polyglot (the translation library we're using) doesn't support markup inside translations, at least in the way it looked like it was planned to be used. So I initially thought we'd either have to split the string up and apply the markup in the ts file, or maybe just link the entire string. Either way, was going to kick that piece down the road.

But thanks for pushing on this, I did some more digging and there's a fairly simple workaround that I added. The link still doesn't point to anything yet.

File: sites/public/layouts/application.tsx [hunk 1]
```diff
@@ -33,7 +34,7 @@ const Layout = (props) => {
         <SiteHeader
           skip={t("nav.skip")}
           logoSrc="/images/detroit-logo.png"
-          notice="This is a preview of our new website. We're just getting started. We'd love to get your feedback."
+          notice={<div dangerouslySetInnerHTML={getTranslationMarkup("nav.getFeedback")}></div>}
```
Comment (willrlin) [inline sites/public/layouts/application.tsx:37]: Is this really dangerous?

**Review by willrlin — APPROVED** [review]

File: sites/public/layouts/application.tsx (hunk 1 above)
Comment (abbiefarr) [reply sites/public/layouts/application.tsx:37]: I don't believe so, based on [the documentation](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml) and [this article](https://pragmaticwebsecurity.com/articles/spasecurity/react-xss-part2.html) I think that this is only dangerous if you're using unsafe (user generated) html.

**Pull request merged**